hydra-node/exe/hydra-node/Main.hs

```
@@ -25,9 +25,9 @@ main = do
   command <- parseHydraCommand
   case command of
     Run options ->
-      run (identifyNode options) `catch` (die . explain)
+      run (identifyNode options) `catch` \(SomeException e) -> die $ displayException e
```

Do we need to catch all exceptions here? Perhaps it would be better to use `catches` to specify all exceptions we know that can happen and and provide slightly better user feedback and then have one handler for `SomeException`.

---

I see you use `displayException` to do pretty much the same thing :+1: